tox.ini

```diff
@@ -3,16 +3,21 @@ recreate = true
 envlist = py-unittest
 indexserver =
     preinstall = https://pypi.python.org/simple
+requires=tox-conda
 
 [base]
 deps =
     :preinstall: -rrequirements.txt
     -rcompanion.txt
-    mkl
+    mkl;'arm' not in platform_machine
 
 [testenv]
 allowlist_externals = bash
-passenv:LAL_DATA_PATH
+passenv=LAL_DATA_PATH
+conda_deps=
+    lalsuite
+    openssl=1.1
+conda_channels=conda-forge
```

Comment:
Can you explain how installing lalsuite through conda actually works? I look at this and would naively think this should actually fail when it tries to

```
 pip install -r requirements. 
 ```

 Does it advertise itself as a python package like the wheels? Otherwise, how would the pycbc install not fail due to the requirements preinstallation which also includes the lalsuite wheel (which I think doesn't exist for arm64)?

Reply:
@spxiwh I took a look at this myself to see if I could understand. I think it may actually be installed as a python package in conda, so maybe that would indeed explain how it could work. 

e.g. https://github.com/conda-forge/lalsuite-feedstock/tree/master/recipe

It certainly suggests a python package like env and not a straight c-style lalsuite install.

Reply:
I don't claim to understand this, but `pip install lalsuite` will not work on ARM. However  `conda install lalsuite` does. (https://github.com/spxiwh/PyCBC_on_ARM/blob/main/README.md)